spec/rubocop/cop/rails/active_record_literal_condition_keys_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::ActiveRecordLiteralConditionKeys, :config do
+  let(:cop_config) { { 'EnforceForEnabled' => true } }
+
+  it 'does not register an offense when using a condition methods are used safely' do
+    RuboCop::Cop::Rails::ActiveRecordLiteralConditionKeys::RESTRICT_ON_SEND.each do |meth|
+      expect_no_offenses("Model.#{meth}(:literal)")
+    end
+  end
+
+  it 'registers an offense when using a condition methods are used unsafely' do
+    RuboCop::Cop::Rails::ActiveRecordLiteralConditionKeys::RESTRICT_ON_SEND.each do |meth|
+      expect_offense(<<~RUBY)
+        Model.#{meth}(unsafe)
+        #{'^' * meth.to_s.length}^^^^^^^^^^^^^^ Avoid using non-literal keys in ActiveRecord conditions.
```

Comment:
Can you use kwargs for `expect_offense` and `^{keyword}`? And can you replace `meth` with` method`?

```diff
--- a/spec/rubocop/cop/rails/active_record_literal_condition_keys_spec.rb
+++ b/spec/rubocop/cop/rails/active_record_literal_condition_keys_spec.rb
@@ -10,10 +10,10 @@ RSpec.describe RuboCop::Cop::Rails::ActiveRecordLiteralConditionKeys, :config do
   end

   it 'registers an offense when using a condition methods are used unsafely' do
-    RuboCop::Cop::Rails::ActiveRecordLiteralConditionKeys::RESTRICT_ON_SEND.each do |meth|
-      expect_offense(<<~RUBY)
-        Model.#{meth}(unsafe)
-        #{'^' * meth.to_s.length}^^^^^^^^^^^^^^ Avoid using non-literal keys in ActiveRecord conditions.
+    RuboCop::Cop::Rails::ActiveRecordLiteralConditionKeys::RESTRICT_ON_SEND.each do |method|
+      expect_offense(<<~RUBY, method: method)
+        Model.#{method}(unsafe)
+        ^^^^^^^{method}^^^^^^^^ Avoid using non-literal keys in ActiveRecord conditions.
       RUBY
     end
   end
```